src/openms/include/OpenMS/FORMAT/MzTabFile.h

```diff
@@ -68,6 +68,25 @@ namespace OpenMS
     // store MzTab file
     void store(const String& filename, const MzTab& mz_tab) const;
 
+    // stream IDs to file
+    void store(
```

Comment:
would be great if our ID and feature datastructures would also provide a stream reader interface ...